web/src/test/resources/custom-dataset.json

```diff
@@ -0,0 +1,5093 @@
+{
```

Comment:
Do we really need a test file this big? :)

Reply:
Reduced the size of the test file in [dc81111](https://github.com/cBioPortal/cbioportal/pull/10039/commits/dc81111a116baeba7d394db47d714a21e3bf6376)